### ISSUE TITLE
Dafang on Synology no recording

### ISSUE THREAD

**Issue by sth313** [EliasKotlyar/Xiaomi-Dafang-Hacks#465]
I successfully added my Dafang to my Synology Surveillance Station. It does capture video, these files are 1 second and always 4mb in size and with a thumbnail. But the records are not playable, with the following error message: the selected record have already been deleted. Why the capture doesn't work?

**Comment by zrenx** [comment]
I've seen the same issue here, turns out I have to turn off the audio on Synology side:
Edit Camera <DAFANG> -> Device Settings -> Video -> Audio Format -> [Disable]

**Comment by sth313** [comment]
Works. Thanks

**Comment by zrenx** [comment]
@sth313  I've managed to get audio in Surveillance Station for both recording and live view.
Here's what I did:
1, Update Dafang Hacks (version 2018-08-09)
2, Dafang settings -> Audio settings -> select audio format "PCM". (in & out sample rate 8000) 
3, In Surveillance Station IP Camera -> Live View Settings -> select both live view and mobile video source "From Surveillance Station"

The last step is to avoid crash of RTSP service when I try to get a live view on my phone with DS cam.
